modules/lvgl/CMakeLists.txt

```diff
@@ -14,6 +14,7 @@ zephyr_library()
 
 zephyr_include_directories(${LVGL_DIR}/src/)
 zephyr_include_directories(.)
+zephyr_include_directories(include)
```

Comment:
how about moving the existin includes in `include` before adding new ones? odd to have both

Reply:
Moved the existing headers to the folder, for now as a separate commit, as it's not necessary for the encoder-input patch. Can squash that if desired :^)